apps/src/storage/dataBrowser/dataVisualizer/VisualizerModal.jsx

```diff
@@ -33,7 +33,7 @@ const styles = {
   },
   chartArea: {
     flexGrow: 1,
-    overflow: 'hidden'
+    overflow: 'auto'
```

Comment:
Setting `overflow: hidden` here was the key change that fixed chart resizing in my last PR: Specifically the case where you shrink the window vertically.  Can you confirm that switching to `overflow: auto` doesn't regress that behavior?  (I suspect it won't, but want to make sure.)

Reply:
Looks like vertical resizing still works with all chart types
![Apr-22-2020 12-30-04](https://user-images.githubusercontent.com/8787187/80025187-16754880-8495-11ea-9e81-fbe13782cf34.gif)